### PR TITLE
First batch of corrections and cleanup.

### PR DESCRIPTION
**Pull request by gheber**
Action items from 2024-08-09 meeting.

### CODE REVIEW

File: documentation/index.md [hunk 1]
```diff
@@ -4,42 +4,25 @@ redirect_from:
...
+* [User Guide](https://support.hdfgroup.org/releases/documentation/hdfview/UsersGuide/index.html)
 
-## HDF4 
-* [Reference Manual](/documentation/hdf4-docs/HDF4_Reference_Manual.pdf)
-* [User Guide](/documentation/hdf4-docs/HDF4_Users_Guide.pdf)
-* [HDF4 Specification and Developers Guide](https://github.com/HDFGroup/hdf4doc/blob/master/DSpec/html_FM/DS.pdf) 
-* [Build and Install HDF4 Applications with CMake](https://raw.githubusercontent.com/HDFGroup/hdf4/master/release_notes/USING_HDF4_CMake.txt)
-* [Build and Install HDF4 C, C++, Fortran Libraries and tools with CMake](https://raw.githubusercontent.com/HDFGroup/hdf4/master/release_notes/INSTALL_CMake.txt)
-* [HDF 4.2 to 4.3 Migration Guide](https://raw.githubusercontent.com/HDFGroup/hdf4/master/doc/HDF-4.2-to-4.3-migration.md) 
+## hdf5-json
```
Comment (ajelenak) [inline documentation/index.md:21]: We chose to use "HDF5/JSON".

File: documentation/index.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ redirect_from:
 
 ## HDFView 
 
-* [User Guide](https://support.hdfgroup.org/releases/documentation/hdfview/UsersGuide/index.html)
+* [User Guide](/releases/documentation/hdfview/UsersGuide/index.html)
```
Comment (byrnHDF) [inline documentation/index.md:19]: Should this be:
/releases/HDFVIEW/v3_3/v3_3_2/documentation/UsersGuide/index.html

File: documentation/index.md [hunk 3]
```diff
@@ -4,42 +4,25 @@ redirect_from:
...
-
-## HDF5 Library and Tools 
-* [Documentation - 1.14.4](https://docs.hdfgroup.org/hdf5/v1_14/index.html)
-* [Documentation - 1.12.1](https://docs.hdfgroup.org/hdf5/v1_12/index.html)
-* [Documentation - 1.10.9](https://docs.hdfgroup.org/hdf5/v1_10/index.html)
-* [Documentation - 1.8.23](https://docs.hdfgroup.org/hdf5/v1_8/index.html)
-* [Registered Virtual File Drivers (VFDs)](/documentation/hdf5-docs/registered_virtual_file_drivers_vfds.html)
-* [Registered VOL Connectors](/documentation/hdf5-docs/registered_vol_connectors.html)
+## HDF5 Library, Tools, and Extensions
+* [Documentation](https://support.hdfgroup.org/documentation/hdf5/index.html) - User Guide, Reference Manual, and other documentation
```
Comment (byrnHDF) [inline documentation/index.md:10]: https://support.hdfgroup.org/releases/HDF5/v1_14/v1_14_4/documentation/doxygen/index.html

File: documentation/index.md (hunk 1 above)
Comment (gheber) [reply documentation/index.md]: Done.

File: documentation/index.md (hunk 2 above)
Comment (gheber) [reply documentation/index.md]: Dones

File: documentation/index.md (hunk 3 above)
Comment (gheber) [reply documentation/index.md]: Done.

File: downloads/index.md [hunk 4]
```diff
@@ -5,25 +5,21 @@ redirect_from:
...
-This page is a temporary replacement for our download archive. We will be working to organize and extend this archive as quickly as possible. 
+# HDF Software Downloads 
 
 ## HDF5 Library and Tools
 
 | Version | Usage | 
 | ---- | ----| 
-| [HDF5 1.14.4](/hdf5/hdf5_1_14_4.md) | This should be the default choice for new users. | 
-| [HDF5 1.12.3](/hdf5/hdf5_1_12_3.md) | This is the last release for HDF5 1.12. Users should move to HDF5 1.14. | 
+| [HDF5 1.14.4](/hdf5/hdf5_1_14_4.md) | Most recent HDF5 library version. |
```
Comment (lkurz) [inline downloads/index.md:14]: These links have been OK when built as github pages, but may need to change ".md" to ".html" .  Or may need to construct the link with the link tag.
e.g. {% link news/index.html %}
Leaving as is until change is necessary.

**Review by loricooperhdf — APPROVED** [review]

**Review by byrnHDF — APPROVED** [review]

**Pull request merged**